### ISSUE TITLE
Window modifier has unexpected background

### ISSUE THREAD

**Issue by PhilipDukhov** [nathantannar4/Transmission#29]
After updating to **1.1.0**, window has white background that hovers whole screen.

In **1.0.3** and earlier, modifier destination was displayed without background.
```
@State var isPresented = false

var body: some View {
    Button("show") {
        isPresented = true
    }
    .window(
        level: .overlay,
        transition: .move(edge: .top).combined(with: .opacity),
        isPresented: $isPresented
    ) {
        Text("hey")
    }
}
```

**Comment by vpontis** [comment]
This is fixed by #30

**Comment by nathantannar4** [comment]
Fixed in `1.1.2`
